docker/test/integration/features/MiNiFi_integration_test_driver.py

```diff
@@ -53,9 +53,9 @@ def __init__(self, context, feature_id: str):
         self.cluster.set_directory_bindings(self.docker_directory_bindings.get_directory_bindings(self.feature_id), self.docker_directory_bindings.get_data_directories(self.feature_id))
         self.root_ca_cert, self.root_ca_key = make_ca("root CA")
 
-        minifi_client_cert, minifi_client_key = make_client_cert(common_name=f"minifi-cpp-flow-{self.feature_id}",
-                                                                 ca_cert=self.root_ca_cert,
-                                                                 ca_key=self.root_ca_key)
+        minifi_client_cert, minifi_client_key = make_cert_without_extended_usage(common_name=f"minifi-cpp-flow-{self.feature_id}",
```

Comment:
I think it would be more intuitive to refer to this as making server cert/key pair, and the `make_client_cert` could remain for client certificate generation.

Reply:
We already have a `make_server_cert` that has a `serverAuth` extended usage, so I would not mix it up with this version where we have no extended usage set.